tests/test-utilities/mocks/cluster.ts

```diff
@@ -1,6 +1,8 @@
 import type { Page } from '@playwright/test';
 import type { GetClusterInfoResponse as Cluster } from '$src/lib/types';
 
+export const CLUSTER_API = '**/api/v1/cluster?';
+
 const MOCK_CLUSTER: Cluster = {
```

Comment:
One thing that is going to be a struggle is remembering the default cluster server version (1.19.3) vs. what you need to pass in for the version for mockClusterApi. Like do we keep updating the default cluster to most recent or most stable (1.20 vs. 1.21) or do we keep it here forever and explicitly pass in newer versions. I think the latter, but it might be a bit of a headache down the road. Just something to think about.

Reply:
Yeah that's a good point. It's possible we'd even want to run the same tests against diff cluster versions, so passing in an override will probably be the way to go.